### PR TITLE
Update UG ladd

### PR DESCRIPTION
**Pull request by angkoonhwee**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ _text_ | Caption for images.
 
    * **`delete`**`3` : Deletes the 3rd student shown in the current list.
 
-   * **`ladd 1 `** : Adds a lesson.
+   * **`ladd 7 date/20 Dec 2021 time/1100-1200 subject/History rates/35`** : Adds a lesson that starts on 20 Dec 2021 for the 7th student shown in the current list.
```
Comment (lingshanng) [inline docs/UserGuide.md:80]: can you make only ladd bold

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1184,13 +1184,13 @@ Format: `redo`
 Example:
 1. Valid Redo command
    1. `edit 1 n/Joe Doe` modifies name of the first student.
-   2. `undo` undoes the modification. Name of first student returns to original name before `edit` command.
+   2. `undo` undoes the modification. Name of the first student returns to original name before `edit` command.
```
Comment (eeliana) [inline docs/UserGuide.md:1187]:
```suggestion
   2. `undo` undoes the modification. Name of the first student returns to the original name before `edit` command.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -1184,13 +1184,13 @@ Format: `redo`
...
    3. `day` displays the calendar for today. This command **does not modify any data**.
    4. `redo` redoes the edit command. The name of the first student will be `Joe Doe` now.
    
 2. Invalid Redo command
    1. `edit 1 n/Joe Doe` modifies name of the first student.
-   2. `undo` undoes the modification. Name of first student returns to original name before `edit` command.
+   2. `undo` undoes the modification. Name of the first student returns to original name before `edit` command.
```
Comment (eeliana) [inline docs/UserGuide.md]:
```suggestion
   2. `undo` undoes the modification. The name of the first student returns to the original name before `edit` command.
```

File: docs/UserGuide.md [hunk 4]
```diff
@@ -966,9 +966,9 @@ Examples:
 
 #### Editing a lesson's outstanding fees: `ledit`
 
-Edits the outstanding fees to specific lesson. Uses the Lesson Edit command as seen in [Editing a lesson](#editing-a-lesson-ledit).
+Edits the outstanding fees to a specific lesson. Uses the Lesson Edit command as seen in [Editing a lesson](#editing-a-lesson-ledit).
```
Comment (Xiaoyunnn) [inline docs/UserGuide.md:969]:
```suggestion
Edits the outstanding fees for a specific lesson. Uses the Lesson Edit command as seen in [Editing a lesson](#editing-a-lesson-ledit).
```

File: docs/UserGuide.md [hunk 5]
```diff
@@ -1184,13 +1184,13 @@ Format: `redo`
...
    1. `edit 1 n/Joe Doe` modifies name of the first student.
-   2. `undo` undoes the modification. Name of first student returns to original name before `edit` command.
+   2. `undo` undoes the modification. The name of the first student returns to original name before `edit` command.
    3. `day` displays the calendar for today. This command **does not modify any data**.
    4. `redo` redoes the edit command. The name of the first student will be `Joe Doe` now.
    
 2. Invalid Redo command
    1. `edit 1 n/Joe Doe` modifies name of the first student.
-   2. `undo` undoes the modification. Name of first student returns to original name before `edit` command.
+   2. `undo` undoes the modification. The name of the first student returns to original name before `edit` command.
```
Comment (eeliana) [inline docs/UserGuide.md:1193]:
```suggestion
   2. `undo` undoes the modification. The name of the first student returns to the original name before `edit` command.
```

File: docs/UserGuide.md [hunk 6]
```diff
@@ -1184,13 +1184,13 @@ Format: `redo`
 Example:
 1. Valid Redo command
    1. `edit 1 n/Joe Doe` modifies name of the first student.
-   2. `undo` undoes the modification. Name of first student returns to original name before `edit` command.
+   2. `undo` undoes the modification. The name of the first student returns to original name before `edit` command.
```
Comment (eeliana) [inline docs/UserGuide.md:1187]:
```suggestion
   2. `undo` undoes the modification. The name of the first student returns to the original name before `edit` command.
```

**Review by eeliana — APPROVED** [review]

**Pull request merged**